src/universities.ts

```diff
@@ -598,6 +609,7 @@ export const universities = [
   'Technische Universität München',
   'Technische Universität München - Campus Weihenstephan',
   'Technische Universität München in Garching',
+  'TH Köln',
```

Comment:
Would it be more appropriate to also have "Technische Hochschule Köln" here instead of "TH Köln" as all the "TU" are also unabbreviated as "Technische Universität XXXX"?

Reply:
Or do we even maybe need both? Like "Technische Hochschule Köln (TH Köln)"?

Reply:
Good question. Now that you mention it, we already have `Technische Hochschule Köln` in the list 🤔 
Maybe what you propose is the best `"Technische Hochschule Köln (TH Köln)"`
@lem503 what do you think?

Reply:
I modified some names to include abbreviations. Thanks for the suggestion @kimkwanka!